osm_fieldwork/update_xlsform.py

```diff
@@ -18,7 +21,48 @@
 # Constants
 FEATURE_COLUMN = "feature"
 NAME_COLUMN = "name"
+TYPE_COLUMN = "type"
 SURVEY_GROUP_NAME = "survey_questions"
+DEFAULT_LANGUAGES = {
+    "english": "en",
+    "french": "fr",
+    "spanish": "es",
+    "swahili": "sw",
+    "nepali": "ne",
```

Comment:
🙌😁

If you can translate the other columns into Nepali then feel free to add them! (might help Tokha I imagine)